.github/workflows/ci.yml

```diff
@@ -276,6 +276,15 @@ jobs:
         run: ./scripts/ci-docker.sh '${{ matrix.unity-version }}' '${{ matrix.platform }}${{ matrix.image-suffix }}' '${{ secrets.UNITY_LICENSE_SERVER_CONFIG }}'
         shell: bash
 
+      # Workaround for missing libMonoPosixHelper.so
+      # See https://github.com/getsentry/sentry-unity/pull/1295
+      # TODO: How to get this into the container ahead of time?
```

Comment:
```suggestion
```
We rely on `GameCI`'s docker images.
I'll point this workaround out on the troubleshooting page.